mne/report.py

```diff
@@ -769,6 +769,14 @@ def _add_figs_to_section(self, figs, captions, section='custom',
             warnings.warn('Could not import mayavi. Trying to render '
                           '`mayavi.core.scene.Scene` figure instances'
                           ' will throw an error.')
+        try:
+            scale = float(scale)
```

Comment:
it can also be callable or `None` at this point in the code, do you have a different preferred check?